bioptim/misc/parameters.py

```diff
@@ -92,7 +92,13 @@ def add_or_replace(ocp, parameter):
             val = func(ocp, cx, **penalty.params)
             ObjectiveFunction.ParameterFunction.clear_penalty(ocp, None, penalty)
             ObjectiveFunction.ParameterFunction.add_to_penalty(
-                ocp, None, val, penalty, weight=weight, quadratic=quadratic
+                ocp,
+                None,
+                val,
+                penalty,
+                weight=weight,
+                quadratic=quadratic,
+                target=np.array([penalty.params["target_value"]]),
```

Comment:
I think this is an error as well though

Reply:
Could you confirm that the target default values are intended? I don't even know what "target_value" is (I did not find any reference of this in the code)

Reply:
@Amedeo123